blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderRunImpl.java

```diff
@@ -97,6 +96,21 @@ public Date getEndTime() {
     }
 
     @Override
+    public String getStartTimeString() {
```

Comment:
Maybe declare these as abstract? default implementation returning null can be misleading.